.ko.yaml

```diff
@@ -47,6 +47,7 @@ baseImageOverrides:
   # prow integration test
   k8s.io/test-infra/prow/test/integration/cmd/fakeghserver: gcr.io/k8s-prow/alpine:v20200713-e9b3d9d
   k8s.io/test-infra/prow/test/integration/cmd/fakegitserver: gcr.io/k8s-prow/git:v20220523-6026203ca9
+  k8s.io/test-infra/prow/test/integration/cmd/fakepubsub: google/cloud-sdk:latest
```

Comment:
please ping to a version instead of latest to ensure deterministic outcome

Reply:
Done